docs/userguide/modules/getting-started/pages/installing-service-binding.adoc

```diff
@@ -161,24 +160,30 @@ NAMESPACE: service-binding-operator
 STATUS: deployed
 REVISION: 1
 
-You can check whether the chart is succesfully installed by running the following command
+. Verify that the chart is succesfully installed:
 
 [source,bash]
 ----
 kubectl get pods --namespace service-binding-operator
 ----
 
-[## Testing the  chart]
+.Example output
+[source,terminal]
+NAME                                        READY   STATUS    RESTARTS   AGE
+service-binding-operator-78c6444b4d-kswhk   1/1     Running   0          21s
+
+[##testing-the-chart]
```

Comment:
[#testing-the-chart]
== Testing the chart 

[NOTE]
====
If you are installing the chart on the Amazon Elastic Kubernetes Service (Amazon EKS) cluster, then perform the following steps:

. Modify the `aws-auth` config map:
+
[source,bash]
----
kubectl edit -n kube-system cm/aws-auth 
----
+
. Add `-system:masters` to mapRoles and save.
. After editing the config map, update the EKS `kubeConfig` file:
+
[source,bash]
----
aws eks update-kubeconfig --name <cluster-name>
----
+
. Continue with the following steps.
====

. Create a `my-k-config` secret and specify the required namespace from your `kubeconfig` file:
+
[source,bash]
----
kubectl create secret generic my-k-config --from-file=kubeconfig=<PATH TO YOUR KUBECONFIG> --namespace service-binding-operator
----
+
.Example output
[source,terminal]
----
secret/my-k-config created
----
+
The output verifies that the `my-k-config` secret is created.

. Run the Helm test and specify the namespace if applicable:
+
[source,bash]
----
helm test service-binding-operator-release --namespace service-binding-operator
----
+
.Example output
[source,terminal]
----
NAME: service-binding-operator-release
LAST DEPLOYED: Mon May 16 10:44:53 2022
NAMESPACE: service-binding-operator
STATUS: deployed
REVISION: 1
TEST SUITE:     service-binding-operator-release-test
Last Started:   Mon May 16 11:01:10 2022
Last Completed: Mon May 16 11:01:22 2022
Phase:          Succeeded
----
+
The `Succeeded` phase from the output indicates that the Helm test has run successfully. 

. Verify that the Helm test has run successfully:
+
[source,bash]
----
kubectl get pods --namespace service-binding-operator
----
+
.Example output
[source,terminal]
----
NAME                                    READY   STATUS      RESTARTS   AGE
service-binding-operator-release-test   0/1     Completed   0          4m28s
----
+
The output verifies that you have successfully installed the {servicebinding-title} using a Helm chart and are able to bind your workload to backing services.

. As a safety measure, delete the secret created and specify the namespace if applicable:
+
[source,bash]
----
kubectl delete secret my-k-config  --namespace service-binding-operator
----
+
.Example output
[source,terminal]
----
secret/my-k-config deleted
----
+
The output verifies that the secret you had created is now deleted.

[NOTE]
====
Deleting the secret avoids exposing the secret credentials of the cluster to which you are connected.
====

Reply:
Just copy-paste whatever I wrote above, for L175 to L255

Reply:
I just noticed that the + option in the edit box appears in some places as dots when the comment renders. I am sending the comments in slack so that the asciidoc symbols come out correctly